### ISSUE TITLE
Ability to sort variables by type

### ISSUE THREAD

**Issue by vraj004** [opencor/opencor#571]
Could we please have the ability to sort the variables by whether they are states/intermediates/parameters? 

The color coding really helps to quickly determine the type. It would also be great to sort not just by components, but also to be able to reshuffle into a list of variables in each type.

**Comment by agarny** [comment]
Yes, I agree that this would be nice to have. In fact, there are quite a few things that could be done to improve the listing of parameters (e.g. see issue #492).
